packages/gatsby/cache-dir/static-entry.js

```diff
@@ -103,315 +103,325 @@ export default ({
+
+    postBodyComponents.push(
+      <script
+        key={`script-loader`}
+        id={`gatsby-script-loader`}
+        dangerouslySetInnerHTML={{
+          __html: windowPageData,
+        }}
+      />
+    )
+
+    // Add chunk mapping metadata
+    const scriptChunkMapping = `/*<![CDATA[*/window.___chunkMapping=${JSON.stringify(
+      chunkMapping
+    )};/*]]>*/`
+
+    postBodyComponents.push(
+      <script
+        key={`chunk-mapping`}
+        id={`gatsby-chunk-mapping`}
+        dangerouslySetInnerHTML={{
+          __html: scriptChunkMapping,
+        }}
+      />
+    )
+
+    let bodyScripts = []
+    if (chunkMapping[`polyfill`]) {
+      chunkMapping[`polyfill`].forEach(script => {
+        const scriptPath = `${__PATH_PREFIX__}${script}`
+        bodyScripts.push(
+          <script key={scriptPath} src={scriptPath} noModule={true} />
+        )
       })
-  )
-
-  postBodyComponents.push(...bodyScripts)
-
-  apiRunner(`onPreRenderHTML`, {
-    getHeadComponents,
-    replaceHeadComponents,
-    getPreBodyComponents,
-    replacePreBodyComponents,
-    getPostBodyComponents,
-    replacePostBodyComponents,
-    pathname: pagePath,
-    pathPrefix: __PATH_PREFIX__,
-  })
+    }
+
+    // Filter out prefetched bundles as adding them as a script tag
+    // would force high priority fetching.
+    bodyScripts = bodyScripts.concat(
+      scripts
+        .filter(s => s.rel !== `prefetch`)
+        .map(s => {
+          const scriptPath = `${__PATH_PREFIX__}/${JSON.stringify(s.name).slice(
+            1,
+            -1
+          )}`
+          return <script key={scriptPath} src={scriptPath} async />
+        })
+    )
 
-  const html = `<!DOCTYPE html>${renderToStaticMarkup(
-    <Html
-      {...bodyProps}
-      headComponents={headComponents}
-      htmlAttributes={htmlAttributes}
-      bodyAttributes={bodyAttributes}
-      preBodyComponents={preBodyComponents}
-      postBodyComponents={postBodyComponents}
-      body={bodyHtml}
-      path={pagePath}
-    />
-  )}`
-
-  return html
+    postBodyComponents.push(...bodyScripts)
+
+    apiRunner(`onPreRenderHTML`, {
+      getHeadComponents,
+      replaceHeadComponents,
+      getPreBodyComponents,
+      replacePreBodyComponents,
+      getPostBodyComponents,
+      replacePostBodyComponents,
+      pathname: pagePath,
+      pathPrefix: __PATH_PREFIX__,
+    })
+
+    const html = `<!DOCTYPE html>${renderToStaticMarkup(
+      <Html
+        {...bodyProps}
+        headComponents={headComponents}
+        htmlAttributes={htmlAttributes}
+        bodyAttributes={bodyAttributes}
+        preBodyComponents={preBodyComponents}
+        postBodyComponents={postBodyComponents}
+        body={bodyHtml}
+        path={pagePath}
+      />
+    )}`
+
+    return { html, unsafeBuiltinsUsage: global.unsafeBuiltinUsage }
+  } catch (e) {
+    e.unsafeBuiltinsUsage = global.unsafeBuiltinUsage
+    throw e
```

Comment:
did anything change except this?

Reply:
I suggest checking with "hide whitespace changes" ( https://github.com/gatsbyjs/gatsby/pull/29560/files?w=1#diff-4cd5b620539129dfa79254b9cdae3bd9b619426b83b412d2647cfb416730d362 )

This whole file added
```
// for this to work we need this function to be sync or at least ensure there is single execution of it at a time
  global.unsafeBuiltinUsage = []
  ```
  at the start of the function, changed return to return object instead of string and wrapped everything in try/catch to decorate caught error with `unsafeBuiltinUsage` to not "lose" it's usage (even tho probably this is might be not needed, it feels safer to do)